props/saber_fett263_buttons.h

```diff
@@ -285,47 +440,1090 @@ EFFECT(faston); // for EFFECT_FAST_ON
 EFFECT(blstbgn); // for Begin Multi-Blast
 EFFECT(blstend); // for End Multi-Blast
 EFFECT(push); // for Force Push gesture in Battle Mode
+EFFECT(mnum); // menu numbers
+EFFECT(quote); // quote on force effect
+#ifdef FETT263_EDIT_MODE_MENU
+EFFECT(medit); // Edit Mode
+#endif
+
+class GestureControlFile : public ConfigFile {
+public:
+  void SetVariable(const char* variable, float v) override {
+    CONFIG_VARIABLE(gestureon, 1);
+#ifdef SWING_GESTURE
+    CONFIG_VARIABLE(swingon, 1);
+#else
+    CONFIG_VARIABLE(swingon, 0);
+#endif
+    CONFIG_VARIABLE(swingonspeed, FETT263_SWING_ON_SPEED);
+#ifdef TWIST_GESTURE
+    CONFIG_VARIABLE(twiston, 1);
+#else
+    CONFIG_VARIABLE(twiston, 0);
+#endif
+#ifdef THRUST_GESTURE
+    CONFIG_VARIABLE(thruston, 1);
+#else
+    CONFIG_VARIABLE(thruston, 0);
+#endif
+#ifdef STAB_GESTURE
+    CONFIG_VARIABLE(stabon, 1);
+#else
+    CONFIG_VARIABLE(stabon, 0);
+#endif
+#ifdef FETT263_TWIST_OFF
+    CONFIG_VARIABLE(twistoff, 1);
+#else
+    CONFIG_VARIABLE(twistoff, 0);
+#endif
+#ifdef FETT263_POWER_LOCK
+    CONFIG_VARIABLE(powerlock, 1);
+#else
+    CONFIG_VARIABLE(powerlock, 0);
+#endif
+#ifdef FORCE_PUSH_CONDITION
+    CONFIG_VARIABLE(forcepush, 1);
+#else
+    CONFIG_VARIABLE(forcepush, 0);
+#endif
+    CONFIG_VARIABLE(forcepushlen, FETT263_FORCE_PUSH_LENGTH);
+    CONFIG_VARIABLE(lockupdelay, FETT263_LOCKUP_DELAY);
+    CONFIG_VARIABLE(clashdetect, FETT263_BM_CLASH_DETECT);
+#ifdef FETT263_MAX_CLASH
+    CONFIG_VARIABLE(maxclash, FETT263_MAX_CLASH);
+#else
+    CONFIG_VARIABLE(maxclash, 16);
+#endif
+
+}
+  bool gestureon;
```

Comment:
Please add comments that explains what these variables control.

Reply:
comments added